app/controllers/search.js

```diff
@@ -23,11 +23,11 @@ export default Controller.extend({
   planets: false,
   films: false,
   categorys: computed(() => ENV.APP.categories),
```

Comment:
categories

Reply:
Intentionally left there, because there is an computed property, that is already named categories

Reply:
That's a very bad reason.. In that case name them different! I assume they have different purposes (otherwise they should not coexist) so name them accordingly.